src/main/java/com/iridium/iridiumskyblock/listeners/PlayerRespawnListener.java

```diff
@@ -17,8 +17,8 @@ public void onRespawn(PlayerRespawnEvent event) {
             user.getIsland().ifPresent(island -> {
                 if (!island.isInIsland(event.getRespawnLocation())) {
                     event.setRespawnLocation(LocationUtils.getSafeLocation(island.getHome(), island));
+                    PlayerUtils.sendBorder(event.getPlayer(), island);
```

Comment:
this is wrong? if should send the border regardless since either way the respawn location will end up being in the island?

Reply:
You are right, but the way it is now, the border is always sent even if the player doesn't respawn on his island / in the island worlds

Reply:
I tried to fix it, please review 👀